ONSdigital/eq-survey-runner
# Apply welsh translations to lms

### What is the context of this PR?
- Added lms_2 welsh schema

### How to review 
- Needs to be thoroughly tested 
- Make sure all routes are working
- No errors with piped values
- Test switching between languages
- Try with static content as well

### Checklist
